seismicpro/src/refractor_velocity/refractor_velocity.py

```diff
@@ -293,6 +294,41 @@ def from_first_breaks(cls, offsets, times, init=None, bounds=None, n_refractors=
         self.times = times
         return self
 
+    @classmethod
+    def from_file(cls, path, encoding="UTF-8"):
+        """Load a near-surface velocity model from a file.
+
+        The file should define a near-surface velocity model at a given location and have the following structure:
+         - The first row contains names of the coordinates parameters ("name_x", "name_y", "x", "y") and
+        names of the RefractorVelocity parameters ("t0", "x1"..."x{n-1}", "v1"..."v{n}").
+         - The second row contains the corresponding values of the velocity model.
```

Comment:
In the dump below there are parameters, not values. Please unify
```suggestion
         - The second row contains the corresponding parameters of the velocity model.
```

Reply:
using of the parameters word can confuse the reader. "Values of parameters" are here. Or "values" to make it simpler